CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 * **[Fix]** Fix reporting carrier information using new iOS 12 APIs when running on iOS 12+.
 * **[Fix]** Fix a memory leak issue during executing SQL queries.
 
+### AppCenterCrashes
```

Comment:
The arm64e slice is there for all modules, maybe we should add lines to other modules as well?

Reply:
Good point.